scripts/example.jl

```diff
@@ -2,26 +2,27 @@
 using FreeBird
 
 # First, we need to read the walkers from a file
+# Note that the periodic boundary conditions are updated to True, True, False
```

Comment:
Does "False" mean they can have any z value or they're confined, by an elastically reflecting wall, to a container?

Reply:
Confined by a reflecting wall.